WebHostLib/static/assets/faq/common-terms_en.md

```diff
@@ -0,0 +1,91 @@
+# Common Multiworld Terms
+
+There are a lot of common terms used when playing in different game randomizer communities and in multiworld as well.
+This document serves as a lookup for common terms that may be used by users in the community or in various other
+documentation.
+
+## Check
+A check is a common term for when you "check", or pick up, a location. In terms of Archipelago this is usually used for
+when a player goes to a location and sends its item or "checks" the location.
+
+## Item
+Items are what you will receive from the server regardless of what game you are playing. This could be a sword, a stat 
+upgrade, a spell, or any other potential receivable for your game.
+
+## Location
+Locations are where items are placed in your game. Whenever you interact with a location, you or another player will
+then receive an item. A location could be a chest, an enemy drop, a shop purchase, or any other interactable that can
+contain items in your game.
+
+## Slot
+A slot is the player name and number assigned during generation. This is equal to the number of `yaml` setting files
```

Comment:
This is not entirely accurate, just because `yaml` setting files can have multiple yamls now. Maybe a small tweak to remove reference to `files`?

Reply:
I think the original intent here I was planning on covering the case of using a weightedsettings.yaml but decided it was out of scope. will reevaluate this though